package.json

```diff
@@ -6,7 +6,8 @@
     "prepare": "husky install",
     "dev": "webpack serve --config webpack/webpack.dev.config.js",
     "build": "webpack --config webpack/webpack.dev.config.js",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "test": "jest tests/"
```

Comment:
should not specify directory tests/

Unit tests are written in the same directory where the function is present. Update it such that the jest config should search of files ending with test.ts/js and run tests